pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -335,3 +335,10 @@ var DefaultLaggingRangesThreshold = 3 * time.Minute
 // DefaultLaggingRangesPollingInterval is the default polling rate at which
 // lagging ranges are checked and metrics are updated.
 var DefaultLaggingRangesPollingInterval = 1 * time.Minute
+
+var Quantize = settings.RegisterDurationSettingWithExplicitUnit(
+	settings.ApplicationLevel,
+	"changefeed.frontier.timestamp_granularity",
```

Comment:
I'd prefer `changefeed.resolved_timestamp.granularity` since ultimately this will be surfaced to the user via quantized resolved timestamps. WDYT?

Reply:
Done.